test/WebSites/ResponseCacheWebSite/Controllers/HomeController.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace ResponseCacheWebSite
```

Comment:
Is it interesting to test having ResponseCache filters on the controller and on actions? (this is a supported scenario). Especially with the append semantics we take now, the filter has now, it might result in conflicting behavior.